src/tools/linkchecker/main.rs

```diff
@@ -23,6 +23,31 @@ use std::rc::Rc;
 
 use crate::Redirect::*;
 
+// Add linkcheck exceptions here
+// If at all possible you should use intra-doc links to avoid linkcheck issues. These
+// are cases where that does not work
+const LINKCHECK_EXCEPTIONS: &[(&str, &[&str])] = &[
+    // These are methods on slice, and `Self` does not work on primitive impls
+    // in intra-doc links (intra-doc links are weird)
```

Comment:
😆

Reply:
I actually meant to say "primitive impls are weird" lol